testsuite/python/analyze_mass_related.py

```diff
@@ -60,17 +65,41 @@ def i_tensor(self, ids):
                  - np.outer(p.pos - com, p.pos - com))
         return I
 
-    def test(self):
+    def test_itensor(self):
         # Particles of type 0
-        I0 = self.i_tensor(range(0, 10, 1))
+        I0 = self.i_tensor(self.system.part.select(
+            lambda p: (not p.virtual) and p.type == 0).id)
```

Comment:
this only works ```IF VIRTUAL_SITES```, *i.e.* ```espressomd.has_features("VIRTUAL_SITES")```

Reply:
the `virtual` property of particles now return False if `VIRTUAL_SITES` is not compiled in